src/Effects.js

```diff
@@ -57,6 +58,34 @@ module.exports = class Effects {
       }
     };
 
+    this.starz = {
+      starSizes: [10, 9, 8, 7, 6, 5, 4, 3, 2, 1],
```

Comment:
Instead of having the values from 1-10 stored in an array, can you restructure your loop to use a counter? Something like: `for (let size = 10; i > 0; i--) {...}`